src/nvim/api/buffer.c

```diff
@@ -478,6 +479,44 @@ ArrayOf(Dictionary) nvim_buf_get_keymap(Buffer buffer, String mode, Error *err)
   return keymap_array(mode, buf);
 }
 
+/// Gets a list of maps describing buffer-local |user-commands|.
+///
+/// @param  buffer  Buffer handle.
+/// @param  opts  Optional parameters. Currently only supports
```

Comment:
Buffer commands are never builtin.  Could accept `builtin=False` as incident of implementation (as implementations are shared), but there is no point to document it as an option here.